framework/wazuh/rbac/orm.py

```diff
@@ -2667,6 +2861,45 @@ def replace_rule_role(self, rule_id: int, current_role_id: int, new_role_id: int
 
         return False
 
+    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
+                     to_id: Optional[int] = None):
+        # This is to avoid an error when trying to update default users roles, policies and rules
+        if from_id == WAZUH_USER_ID and to_id == WAZUH_WUI_USER_ID:
+            return
```

Comment:
We should warn users that default resources can't be updated

```suggestion
        # This is to avoid an error when trying to update default users, roles, policies and rules
        if from_id == WAZUH_USER_ID and to_id == WAZUH_WUI_USER_ID:
            return
```

Reply:
Added warnings in 5b3bd4eb01eec22333fb389e29c12a61e7e0c734.